azavea/cac-tripplanner
# Display place card times for events on map page

## Overview

Add start and end date/times to event place cards in map sidebar.


### Demo

![image](https://user-images.githubusercontent.com/960264/67044189-b96d9400-f0f9-11e9-84fd-e689b7234b55.png)


## Testing Instructions

 * View on map an event with dates/times for one or more destinations
 * Dates/times should display in sidebar place cards


Closes #1172.
